ts/src/bitfinex.ts

```diff
@@ -1690,11 +1690,10 @@ export default class bitfinex extends Exchange {
             }
         }
         if (throwError) {
-            const feedback = this.id + ' ' + body;
             const message = this.safeString2 (response, 'message', 'error');
-            this.throwExactlyMatchedException (this.exceptions['exact'], message, feedback);
-            this.throwBroadlyMatchedException (this.exceptions['broad'], message, feedback);
-            throw new ExchangeError (feedback); // unknown message
+            this.throwExactlyMatchedException (this.exceptions['exact'], message, message);
+            this.throwBroadlyMatchedException (this.exceptions['broad'], message, message);
+            throw new ExchangeError (message); // unknown message
```

Comment:
You didn't add the code in the `message` here, is there any specific reason?

Reply:
I should doublecheck but regarding the response structure I assumed that there were no error code

Reply:
confirmed